sigs/addons/MIGRATION_RFC_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Migrate XXXXX from TensorFlow Addons to TensorFlow Core
+
+| Status      | Proposed (Waiting for Sponsor)                                                                                           |
+| :---------- | :------------------------------------------------------------------------------------------------- |
+| **RFC #**   | TBD after PR |                                       |
+| **Authors** | XXXXXXXXXX   |
+| **Sponsor** | XXXXXXXXXX   |
+| **Updated** | YYYY-MM-DD   |
+| **Sponsorship Deadline** | YYYY-MM-DD (45 Days after submission)  |
+
+## Rationale for Migration
+* What are the use cases for the addon?
+* OSS usage, H5 Index, etc.
+
+## Historical Information
```

Comment:
I don't know how it is easy to have a quick query but for stability we could ask how many bugs was handled in the last (x) month(s).